.renovaterc.json

```diff
@@ -4,39 +4,46 @@
     ":enablePreCommit",
     "schedule:weekly"
   ],
-  "ignoreDeps": ["relekang/python-semantic-release"],
-  "pip_requirements": {
-    "fileMatch": ["^requirements(-[\\w]*)?\\.txt$"]
-  },
+  "enabledManagers": [
+    "pip_requirements",
+    "pre-commit"
+  ],
+  "ignoreDeps": [
+    "relekang/python-semantic-release"
+  ],
   "regexManagers": [
     {
-      "fileMatch": ["^tests\\/functional\\/fixtures\\/.env$"],
-      "matchStrings": ["GITLAB_TAG=(?<currentValue>.*?)\n"],
+      "fileMatch": [
+        "^tests\\/functional\\/fixtures\\/.env$"
+      ],
+      "matchStrings": [
+        "GITLAB_TAG=(?<currentValue>.*?)\n"
+      ],
       "depNameTemplate": "gitlab/gitlab-ce",
       "datasourceTemplate": "docker",
       "versioningTemplate": "loose"
-    },
-    {
-      "fileMatch": ["^.pre-commit-config.yaml$"],
-      "matchStrings": ["- (?<depName>.*?)==(?<currentValue>.*?)\n"],
-      "datasourceTemplate": "pypi",
-      "versioningTemplate": "pep440"
     }
   ],
   "packageRules": [
     {
-      "packagePatterns": ["^gitlab\/gitlab-.+$"],
+      "packagePatterns": [
+        "^gitlab\/gitlab-.+$"
+      ],
       "automerge": true
     },
     {
-      "matchPackagePrefixes": ["types-"],
+      "matchPackagePrefixes": [
+        "types-"
+      ],
       "groupName": "typing dependencies"
     },
     {
-        "matchPackagePatterns": ["(^|/)black$"],
-        "versioning": "pep440",
-        "ignoreUnstable": false,
-        "groupName": "black"
-     }
+      "matchPackagePatterns": [
+        "(^|/)black$"
+      ],
+      "versioning": "pep440",
+      "ignoreUnstable": false,
+      "groupName": "black"
+    }
```

Comment:
Since we're doing some cleanup here, I think we can remove the custom black handling entirely. We had this because black was beta for a really long time and updates were ignored.

With https://black.readthedocs.io/en/latest/the_black_code_style/index.html#stability-policy I think we can get rid of this (sorry, you'll have to change this locally, github won't let me suggest changes to lines above):
```suggestion
```

Reply:
Sure. Removing those lines.